app/services/conversation_bot.rb

```diff
@@ -1,13 +1,297 @@
 class ConversationBot
+  NOW_TIME = /now|ahora/i.freeze
 
+  CALL_FOR_HELP = /\Ahelp|ayuda\z/i.freeze
+
+  NUMBER_STRINGS = {
+      0 => [/0/, /zero/, /cero/],
+      1 => [/1/, /one/, /uno/],
+      2 => [/2/, /two/, /dos/],
+      3 => [/3/, /three/, /tres/],
+      4 => [/4/, /four/, /cuatro/]
+  }.freeze
+
+  DONT_KNOW_FRAGMENTS = [/don't know/i, /no se/i, /no sé/i].freeze
```

Comment:
we need to be careful at this point we are asking for an address. if we anchored these texts to the whole input it would be safer, but we don't want to abandon them for wanting to be picked up at 100 dunnor street or in Runsure Ohio. But yes, agree we want to be smart about these expressions. Perhaps one check that would help is no digits present + one of these fragments.